app/queries/servers/file.ts

```diff
@@ -16,3 +19,18 @@ export const getFileById = async (database: Database, fileId: string) => {
         return undefined;
     }
 };
+
+export function observeCanDownloadFiles(database: Database) {
+    const enableMobileFileDownload = observeConfigBooleanValue(database, 'EnableMobileFileDownload');
+    const isLicened = observeLicenseBooleanValue(database, 'IsLicensed');
+    const hasCompliance = observeLicenseBooleanValue(database, 'Compliance');
+
+    const complianceEnabled = combineLatest([isLicened, hasCompliance]).pipe(
+        switchMap(([licensed, compliance]) => of$(licensed || compliance)),
+    );
+
+    const canDownloadFiles = combineLatest([enableMobileFileDownload, complianceEnabled]).pipe(
+        switchMap(([download, compliance]) => of$(compliance && download)),
```

Comment:
I just cross checked other files... I think you can download if : 
1. enableMobileFileDownload is true
2. compliance is false

```suggestion
        switchMap(([download, compliance]) => of$(!compliance ||  download)),
```

Reply:
@larkox  can you please confirm this one?

Reply:
After some research, the idea is the following:
- EnableMobileFileDownload should only be set (or taken into account) if Compliance is set.
- If Compliance is not set, we understand you can always download.

Therefore, the suggestion of @avinashlng1080 should be right.